enterprise/coderd/provisionerdaemons.go

```diff
@@ -55,11 +55,6 @@ func (api *API) provisionerDaemonsEnabledMW(next http.Handler) http.Handler {
 // @Router /organizations/{organization}/provisionerdaemons [get]
 func (api *API) provisionerDaemons(rw http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
-	org := httpmw.OrganizationParam(r)
-	if !api.Authorize(r, rbac.ActionRead, rbac.ResourceProvisionerDaemon.InOrg(org.ID)) {
-		httpapi.Forbidden(rw)
-		return
-	}
```

Comment:
Why are we removing the auth check here?

Reply:
It was redundant. We do an `AuthorizeFilter` on the returned provisioner daemons: https://github.com/coder/coder/blob/64a246dacfe0a0ffce74b477fec83b7204f0ae9d/enterprise/coderd/provisionerdaemons.go#L72

Reply:
It was also incorrect as provisioner daemons have no org scope atm.